duckduckgo/macos-browser
# Add encryption/decryption layer to scheduler helper database operations

## Task

https://app.asana.com/0/1204006570077678/1205259793136461/f

## Description

Implement the mapper and do the correct transformation of encrypting/decrypting and encoding/decoding the data.

## Steps to test

1. Run DBP debug UI and save a profile
2. Close the app, and a breakpoint inside the `testFetchingSchedulerData` debug print
3. Check the data is being returned correctly and unencrypted from the vault
